turbo/shortcuts.py

```diff
@@ -0,0 +1,26 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+def render_turbo(request, template_name, location, context=None, content_type="text/html", status=None, using=None):
+    # A request could be made from 2 places.
+
+    # Either it is a Turbo Frame that initiated the request: we can detect it with
+    # `turbo-stream` in the `Accept` header.
+    if "turbo-stream" in request.META['HTTP_ACCEPT']:
+        # Notify the template that this is a trubo stream response sent over HTTP
+        # (in opposition to turbo steams sent via websockets)
+        context["turbo__is_http"] = True
+
+        # Notify the turbo client that we are sending a turbo-stream message
+        # so the frontend does not try to redirect the user
+        content_type = '%s; turbo-stream;' % content_type
```

Comment:
```suggestion
        content_type = f'{content_type}; turbo-stream;'
```
This is certainly a personal opinion, but I feel like f-strings are a lot more explicit than old-style format strings.

Reply:
It's definitely better and i don't think we want to support python 3.5 so let's use all the modern things :)